twoliter/embedded/rpm2img

```diff
@@ -237,10 +237,22 @@ for kit in "${kits[@]}"; do
     <<<"${INVENTORY_DATA}")"
 done
 
+ALIAS_COUNT="$(jq '[.Content[] | select(.Alias != null and (.Alias | test("inventory-alias")))] | length' <<<"${INVENTORY_DATA}")"
+INVENTORY_DATA="$(jq '
+  .Content |= (
+    . + [
+      .[] |
+      select(.Alias | test("inventory-alias")) |
+      .Name = (.Alias | capture("inventory-alias\\((?<a>[^)]+)\\)") | .a)
+    ]
+  ) |
+  .Content |= map(del(.Alias))
+' <<<"${INVENTORY_DATA}")"
+
 # Verify we successfully inventoried some RPMs.
 INVENTORY_COUNT="$(jq '.Content | length' <<<"${INVENTORY_DATA}")"
 RPM_COUNT="$(find "${PACKAGE_DIR}" -maxdepth 1 -type f -name '*.rpm' | wc -l)"
-if [[ "${INVENTORY_COUNT}" -ne "${RPM_COUNT}" ]]; then
+if [[ "$((INVENTORY_COUNT - ALIAS_COUNT))" -ne "${RPM_COUNT}" ]]; then
```

Comment:
nit: might be more readable to add the RPM and alias counts together and compare with inventory count.